.buildkite/pipeline.yml

```diff
@@ -28,6 +28,54 @@ steps:
 
   - wait
 
+  - block: ':whale: Publish to App Center'
+    prompt: 'Confirm publishing to App Center?'
+
+  - name: ':rocket: Publish demo app to App Center'
```

Comment:
forgot depends_on: 'release-block-appcenter' ?

Reply:
I believe the block step creates an implicit dependancy for the step after it so declaring an explicit dependancy with depends_on isn't necessary (its needed if using input instead of block)